[Go][Bug] dotprompt fails to compile with invopop/jsonschema v0.14.0 due to ordered-map type mismatch

### Context
I encountered this while building a Go project that depends on `github.com/firebase/genkit/go`, which pulls in `github.com/google/dotprompt/go`.

The build resolved:

```text
github.com/google/dotprompt/go v0.0.0-20260227225921-0911cf9ecf0e
github.com/invopop/jsonschema v0.14.0
```

With that module graph, `go build ./...` fails while compiling `github.com/google/dotprompt/go/dotprompt`.

### Expected Behavior
`dotprompt` should compile with the resolved `github.com/invopop/jsonschema` version, or its module requirements should prevent Go from selecting an incompatible `jsonschema` version.

Specifically, `dotprompt` should use the same ordered-map package type that `jsonschema.Schema.Properties` expects.

### Actual Behavior
The build fails with a type mismatch in `dotprompt/picoschema.go`:

```text
# github.com/google/dotprompt/go/dotprompt
.../dotprompt/picoschema.go:201:15: cannot use orderedmap.New[string, *jsonschema.Schema]() (value of type *"github.com/wk8/go-ordered-map/v2".OrderedMap[string, *jsonschema.Schema]) as *"github.com/pb33f/ordered-map/v2".OrderedMap[string, *jsonschema.Schema] value in struct literal
```

`dotprompt` imports:

```go
orderedmap "github.com/wk8/go-ordered-map/v2"
```

and assigns it to:

```go
jsonschema.Schema{
    Properties: orderedmap.New[string, *jsonschema.Schema](),
}
```

But `github.com/invopop/jsonschema v0.14.0` changed `Schema.Properties` to use:

```go
github.com/pb33f/ordered-map/v2
```

instead of:

```go
github.com/wk8/go-ordered-map/v2
```

So `dotprompt` is source-incompatible with `jsonschema v0.14.0`. Downgrading `github.com/invopop/jsonschema` to `v0.13.0` works around the issue.

--- a/go.MODULE.bazel
+++ b/go.MODULE.bazel
@@ -16,7 +16,7 @@
 
 """Configuration for Bazel with Go dependencies."""
 
-GO_VERSION = "1.24.11"
+GO_VERSION = "1.25.12"
 
 bazel_dep(name = "rules_go", version = "0.59.0", repo_name = "io_bazel_rules_go")
 

--- a/go.MODULE.bazel
+++ b/go.MODULE.bazel
@@ -38,7 +38,7 @@ use_repo(
     "com_github_google_go_cmp",
     "com_github_invopop_jsonschema",
     "com_github_mbleigh_raymond",
+    "com_github_pb33f_ordered_map_v2",
     "com_github_smacker_go_tree_sitter",
-    "com_github_wk8_go_ordered_map_v2",
     "org_golang_x_text",
 )

--- a/go/dotprompt/BUILD.bazel
+++ b/go/dotprompt/BUILD.bazel
@@ -35,7 +35,7 @@ go_library(
         "@com_github_goccy_go_yaml//:go-yaml",
         "@com_github_invopop_jsonschema//:jsonschema",
         "@com_github_mbleigh_raymond//:raymond",
-        "@com_github_wk8_go_ordered_map_v2//:go-ordered-map",
+        "@com_github_pb33f_ordered_map_v2//:ordered-map",
         "@org_golang_x_text//unicode/norm",
     ],
 )
@@ -59,6 +59,6 @@ go_test(
         "@com_github_google_go_cmp//cmp/cmpopts",
         "@com_github_invopop_jsonschema//:jsonschema",
         "@com_github_mbleigh_raymond//:raymond",
-        "@com_github_wk8_go_ordered_map_v2//:go-ordered-map",
+        "@com_github_pb33f_ordered_map_v2//:ordered-map",
     ],
 )

